src/components/Scheme/SchemeContainer.tsx

```diff
@@ -53,6 +55,28 @@ const mapDispatchToProps = {
 
 class SchemeContainer extends React.Component<IProps, null> {
 
+  private getSchemeIsActive(scheme: ISchemeState): boolean {
+    if (scheme.name === "SchemeRegistrar") {
+      /**
+       * If SchemeRegistrar has only one or the other active, then we'll deal with that case in
```

Comment:
it would be clearer if the "isSchemeActive"  logic is all in one single place - and change the signature to something like `getSchemeisActive(scheme, action)`

Reply:
This comment is confusing because that *is* the name of the function.

Reply:
Done